module/actor/actor.mjs

```diff
@@ -1064,6 +1064,10 @@ export class SWSEActor extends Actor {
         })
     }
 
+    // getter for further speed calculations
+    get resolvedSpeed() {
+        return parseInt(/([\w\s]*)\s(\d*)/.exec(this.speed)[2]);
```

Comment:
this will work fine when there is a single speed, if there are multiple speeds, it will select the first one.  vehicles will also not work.  let's name it "numericSpeed".  does the plugin you're trying to support accept a flying speed for instance?

Reply:
Thanks for the review!

The plugin expects a single attribute to calculate with its integer value the allowed movement distance, e.g. actor.speed or actor.numericSpeed in this case. Best approach would be of course to switch the numeric speed to the number that applies best for the token. If it is flying at the moment the numericSpeed getter should return the flight speed for example.

Reply:
good call.  feel free to leave this as is and log an enhancement issue.  i think selecting movement type might be a little out of scope for your fix

Reply:
https://github.com/kypvalanx/Foundry-VTT-StarWars-SagaEdition/issues/423 created